CNAME

```diff
@@ -0,0 +1 @@
+docs.metasploit.com
```

Comment:
Consider moving file to `docs/CNAME` as I believe this file needs to be at the root on the `gh_pages` branch used as source for the static page see: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

> Under "Custom domain", type your custom domain, then click Save. This will create a commit that adds a CNAME file in the root of your publishing source.

https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites

> The default publishing source for project sites is the root of the `gh-pages` branch.

Reply:
The `gh-pages` automation would need to be updated to ensure this file is at the root for that branch weekly.